src/dom.js

```diff
@@ -605,3 +605,13 @@ export function tryFocus(element) {
     // IE <= 7 may throw exceptions when focusing on hidden items.
   }
 }
+
+/**
+ * Whether the given window is in an iframe or not.
+ * @param {!Window} win
+ * @return {boolean}
+ */
+export function isIframed(win) {
```

Comment:
Yep. And I think this might not be in Viewer. We should have it somewhere else, as it is a doc level information.